utils/src/main/kotlin/Utils.kt

```diff
@@ -245,14 +245,24 @@ fun normalizeVcsUrl(vcsUrl: String): String {
         return File(url).toSafeURI().toString()
     }
 
-    if (uri.scheme != "ssh" && uri.host != null && uri.host.endsWith("github.com")) {
-        // Ensure the path ends in ".git".
-        val path = if (uri.path.contains(Regex("\\.git(/|$)"))) uri.path else uri.path + ".git"
-
-        // Remove any user name and "www" prefix.
-        val host = uri.authority.substringAfter("@").removePrefix("www.")
-
-        return "https://" + host + path
+    // Handle host-specific normalizations.
+    if (uri.host != null) {
+        when {
+            uri.host.endsWith("github.com") -> {
```

Comment:
`if (uri.host != null && uri.host.endsWith("github.com"))`

Reply:
Not "soon", but I wanted to emphasize where these would be added, yes.